uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -33,7 +33,7 @@ float16 static_air_temperature          # Kelvin
 float16 pitot_temperature               # Kelvin
 
 
-float16[<=16] covariance                # order of diagonal elements : 
+float16[<=25] covariance                # order of diagonal elements : 
                                         # static_pressure, differential_pressure,
                                         # static_air_temperature, pitot_temperature
```

Comment:
1. static_pressure
2. differential_pressure
3. static_air_temperature
4. pitot_temperature

4*4 = 16

Seems legit. The sensor temperature variances were omitted intentionally per the discussion we had.

What is the fifth variance you think we're missing here?